cli/tools/task.rs

```diff
@@ -96,26 +126,30 @@ struct TaskRunner<'a> {
 }
 
 impl<'a> TaskRunner<'a> {
-  async fn run_task(
+  async fn run_tasks(
     &self,
-    task_name: &String,
+    task_names: Vec<String>,
   ) -> Result<i32, deno_core::anyhow::Error> {
-    let Some((dir_url, task_or_script)) = self.tasks_config.task(task_name)
-    else {
-      if self.task_flags.is_run {
-        return Err(anyhow!("Task not found: {}", task_name));
+    // TODO(bartlomieju): we might want to limit concurrency here - eg. `wireit` runs 2 tasks in parallel
+    // unless and env var is specified.
+    // TODO(marvinhagemeister): Run in sequence for now as it's not safe
+    // to run all tasks in parallel. We can only run tasks in parallel where
+    // all dependencies have been executed prior to that.
```

Comment:
I think we should resolve this before landing because otherwise we'll switch to parallel in the future and break stuff.

Reply:
I'm going to look into fixing this.

Reply:
I managed to do it after fighting rust all evening. Tasks are now run in parallel if possible.

Reply:
Great! I think one last thing here is to run at most X tasks in parallel that equals to no of CPUs on the machine and respect the `DENO_JOBS` env var. I can look into that tomorrow though.